ansible/roles/dhcrelay/templates/etc/ansible/facts.d/dhcrelay.fact.j2

```diff
@@ -0,0 +1,28 @@
+#!{{ ansible_python['executable'] }}
+
+# Copyright (C) 2020 CipherMail B.V. <https://www.ciphermail.com/>
+# Copyright (C) 2020 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# {{ ansible_managed }}
+
+from json import dumps
+import os
+import subprocess
+
+
+def cmd_exists(cmd):
+    return any(
+        os.access(os.path.join(path, cmd), os.X_OK)
+        for path in os.environ['PATH'].split(os.pathsep)
+    )
+
+
+output = {'installed': cmd_exists('dhcrelay')}
+
+if output['installed']:
+    output['version'] = subprocess.check_output(
+        ['dhcrelay', '--version'], stderr=subprocess.STDOUT
```

Comment:
Ah, so that's the source of the role name... Sounds good then. I would go with `dhcp_relay`, but if the command line name exists, that's fine with me. :+1:

Reply:
Yeah, naming things is one of the two hard things in IT, next to cache invalidation and off-by-one errors :)
I did indeed name this role dhcrelay after the command line name, just as with dhcpd. It's also a bit shorter than dhcp_relay and maybe less ambiguous (there could be more software out there that does DHCP relaying).